imager/views.py

```diff
@@ -0,0 +1,19 @@
+from django.http import HttpResponse, HttpResponseRedirect, Http404
+from django.template import RequestContext, loader
+from django.shortcuts import render
+
+
+def stub(request, *args, **kwargs):
```

Comment:
@markableidinger it's a stub view we used to explore the relationship of url capture parameters to the function signatures of view functions.  It's not an abomination, it's a useful pedagogical tool.  However, it doesn't need to be here any more, and can be removed now that the app is working with real views.